### PR TITLE
fix a few typos and improve phrasing in GOALS

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: GOALS.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 The ultimate goal is to make the development of apps adhering to [WAI-ARIA](http://www.w3.org/TR/wai-aria/) a breeze. The ultimatest™ goal is to make this library obsolete by lobbying the W3C and browser vendors into supporting the basic tools provided by ally.js. (One may dream…)
 
-Applications have to deal with quite a number of different issues to get to a state we may call "accessible". There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/) to provide help the browser provide the operating system's Accessibility Infrastructure with the necessary details.
+Applications have to deal with quite a number of different issues to get to a state we may call "accessible." There's the issue of *maintaining the focus*, a subset of providing *keyboard navigation*. Then there's [WAI-ARIA](http://www.w3.org/TR/wai-aria/), which helps the browser provide the operating system's Accessibility Infrastructure with the necessary details.
```
Comment (cvan) [inline GOALS.md]: I'm not sure exactly what that last sentence was trying to say. but I think I got it. let me know.

File: GOALS.md [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 
-It does not make sense to implement the same things over and over again, which is why `ally.js` is designed in a way that while allowing simple and drect integration (like jQuery), exposes every function individually. The modularity is paramount for preventing application and library authors from copy-pasting code, or worse, rolling their own flawed implementation. While ally.js is not perfect (it's software, it's written by me, …), I want to make this *the place we focus our efforts* - much like has happened for the DOM by way of jQuery.
+It does not make sense to implement the same things over and over again, which is why `ally.js` is designed in a way that, while allowing simple and direct integration (like jQuery), exposes every function individually. The modularity is paramount for preventing application and library authors from copy-pasting code or, worse, rolling their own flawed implementations. While ally.js is not perfect (it's software, it's written by me, …), I want to make this *the place we focus our efforts* - much like has happened for the DOM by way of jQuery.
```
Comment (cvan) [inline GOALS.md]: the diff doesn't make it clear, but this is what I changed:

`drect` ➔ `direct`
`that while allowing` ➔ `that, while allowing`
`code, or worse, rolling` ➔ `code or, worse, rolling`
`flawed implementation.` ➔ `flawed implementations.`

File: GOALS.md (hunk 1 above)
Comment (rodneyrehm) [inline GOALS.md]: why did you move the dot inside the quotes `"accessible."`?

the last sentence is incomplete. it was supposed to say something about "additional semantics" (necessary details to explain what the element is supposed to mean).

**Comment by rodneyrehm** [comment]
thx! will merge tomorrow morning (starting to record this week's podcast now)

File: GOALS.md (hunk 1 above)
Comment (cvan) [inline GOALS.md]: Per English style guides (MLA, APA, and Chicago), period, comma, question mark, exclamation marks go of inside quotation marks. But colons and semicolons go outside of quotation marks.

American English is funny that way.

<hr>

_Sources:_
- https://owl.english.purdue.edu/owl/resource/577/03/
- http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html
 http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation.html
- http://www.grammarly.com/answers/questions/4726-quotation-marks-and-other-punctuation-inside-or-outside/

Comment (rodneyrehm) [inline GOALS.md]: Interesting. As a non-native, the reflex I get is simply dropping the quotes and avoiding the issue altoghter. But then… why not. Weird language is weird.
